frontend/nuxt.config.ts

```diff
@@ -54,7 +54,10 @@ export default defineNuxtConfig({
 			ENVIRONMENT === 'testnet',
 	},
 	nitro: {
-		preset: 'firebase',
+		// Workaround: Until whatever depends on Nitro has been upgraded to include 'https://github.com/unjs/nitro/commit/92d711fe936fda0ff877c23d8a0d73ed4ea4adc4', we do it manually.
+		// Workaround on workaround: Have to use empty string instead of "node-server" as otherwise we get error "Cannot resolve preset: node-server"
+		// (see 'https://forum.cleavr.io/t/cannot-resolve-node-server-preset/686').
+		preset: process.env.NITRO_PRESET === 'node-server' ? '' : 'firebase',
```

Comment:
Can't build, tried with `NITRO_PRESET=node-server yarn build`, but got the error described in the workaround on workaround comment 😁

Reply:
I must have added the empty string thing after testing because that's what breaks it. The problem is that preset `node-server` was renamed from `server`, and the empty prefix *in the config file* would make it fall back to that value. So just using `NITRO_PRESET=server` is the correct solution for the nuxt/nitro version that we use.